### ISSUE TITLE
Calendar dropdown not accessible 

### ISSUE THREAD

**Issue by dariiing** [cryptpad/cryptpad#1553]
![image](https://github.com/cryptpad/cryptpad/assets/95225431/3d04adc4-6000-4eec-a512-7dcfd9ab7249)
The calendar picker dropdown inside the event modal cannot be accessed using the keyboard. This limitation prevents users who do not use a mouse from selecting different calendars.
